src/main.rs

```diff
@@ -199,10 +199,14 @@ fn git_repos(_options: &Options, _args: &[&OsStr]) -> bool {
 #[cfg(feature = "git")]
 fn get_files_in_dir(paths: &mut Vec<PathBuf>, path: PathBuf) {
     let temp_paths = if path.is_dir() {
-        path.read_dir()
-            .unwrap()
-            .map(|entry| entry.unwrap().path())
-            .collect::<Vec<PathBuf>>()
+        match path.read_dir() {
+            Err(_) => {
+                vec![path]
+            }
+            Ok(d) => d
+                .map(|entry| entry.unwrap().path())
```

Comment:
```suggestion
                 .filter(std::result::Result::is_ok)
                .map(|entry| entry.unwrap().path())
```
maybe throw a check in there while we are at it? I'm surprised that .unwrap was there to begin with. Thanks for picking this up :+1:

Reply:
Added the filter, good suggesting, I should've seen it myself. Too focused on just fixing the issue, I guess.